deploy/docker/start.sh

```diff
@@ -1,53 +1,53 @@
 #!/bin/sh
-set -e
+set -e -u
 
 emqx_exit(){
-    # tail erlang.log.*
-    erlang_log=$(echo $(ls -t /opt/emqx/log/erlang.log.*) | awk '{print $1}')
-    num=$(sed -n -e '/LOGGING STARTED/=' ${erlang_log} | tail -1)
-    tail -n +$((num-2)) ${erlang_log}
+    # At least erlang.log.1 exists
+    if [ -f /opt/emqx/log/erlang.log.1 ]; then
+        # tail emqx.log.*
+        erlang_log=$(echo $(ls -t /opt/emqx/log/erlang.log.*) | awk '{print $1}')
+        num=$(sed -n -e '/LOGGING STARTED/=' ${erlang_log} | tail -1)
+        [ ! -z $num ] && [ $num -gt 2 ] && tail -n +$(expr $num - 2) ${erlang_log}
```

Comment:
sorry for the confusion, I tested it locally, it seems that the number for `tail -n` can be zero or negative.

Reply:
i.e. there is no need for the `[ $num -gt 2 ]` check

Reply:
I tested locally and found that `tail -n` can be zero, but not negative